datafusion/functions-aggregate/src/count.rs

```diff
@@ -79,6 +79,12 @@ pub fn count_distinct(expr: Expr) -> Expr {
     ))
 }
 
+/// Count(*), Count(), Count(1) are all equivalent expression
+/// In DataFusion, we convert them to Count(1) expression
+pub fn count_wildcard() -> Expr {
```

Comment:
wildcard here is a syntactical remnant.

this is a function to call all rows, so call it like that

```suggestion
/// Creates aggregation to count all rows
pub fn count_all() -> Expr {
```